Theme.css

```diff
@@ -2428,7 +2428,7 @@ div#network.network > div > canvas {
 /* from https://github.com/Dunedan/mbp-2016-linux on Windows*/
 /* <g-emoji alias="heavy_check_mark"
     fallback-src="https://assets-cdn.github.com/images/icons/emoji/unicode/2714.png"
-    ios-version="6.0"></g-emoji> */
+    ios-version="6.0">✔️</g-emoji> */
```

Comment:
Can you provide a before and after screen shot of this?

Reply:
This is just a web editor issue that doesn't support special characters because I applied changes from the web editor here.